src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -727,5 +728,24 @@ internal static bool ConnectionExists(IEnumerable<KeyValuePair<string, string>>
                 : null;
             return (new Uri($"{protocol}://0.0.0.0:{Port}"), new Uri($"{protocol}://localhost:{Port}"), cert);
         }
+        private void CheckSettingFunctionWorkerRuntime()
```

Comment:
Check function worker runtime setting for what? Can the method name be more specific of what it is actually doing?

Reply:
Right so it's not just checking, it's taking an action too. `EnsureWorkerRuntimeIsSet` might be a better name here.